packages/node_modules/@cerebral/mobx-state-tree/README.md

```diff
@@ -62,7 +62,7 @@ export default Module({
   // Define computed (receives a value to compute)
   // store.helloFromFoo("someName")
   computed: {
-    helloFrom(name) {
+    helloFromFoo(name) {
```

Comment:
Maybe use better `helloFrom` in comment as you're using it again later in `App.js`.
Above seems also be a mismatch. You have `store.upper` in comment but `upperName() ` as function.